### PR TITLE
clight: 4.1 -> 4.2

### PR DESCRIPTION
**Pull request by eadwu**
###### Motivation for this change
https://github.com/NixOS/nixpkgs/pull/112117

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/clight/clightd.nix [hunk 1]
```diff
@@ -1,23 +1,31 @@
-{ lib, stdenv, fetchFromGitHub
+{ lib, stdenv, fetchFromGitHub, fetchpatch
 , dbus, cmake, pkg-config
-, glib, udev, polkit, libmodule
+, glib, udev, polkit, libusb, libjpeg, libmodule
```
Comment (cole-h) [inline pkgs/applications/misc/clight/clightd.nix:3]: `libusb` is an alias to `libusb1`.

```suggestion
, glib, udev, polkit, libusb1, libjpeg, libmodule
```

File: pkgs/applications/misc/clight/clightd.nix [hunk 2]
```diff
@@ -48,15 +56,18 @@ stdenv.mkDerivation rec {
     glib
     udev
     polkit
+    libusb
```
Comment (cole-h) [inline pkgs/applications/misc/clight/clightd.nix:59]:
```suggestion
    libusb1
```

**Review by cole-h — CHANGES_REQUESTED** [review]

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112254` at 6e1d9046 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/7c16ee9687d18f2688d642629a571673">clight</a></li></li>
    <li><a href="https://gist.github.com/3e9f6393f0348e4cf3c5b35224d6376f">clightd</a></li></li>
  </ul>
</details>
<details>
  <summary>2 warnings produced:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/clight/clightd.nix:81:5:
```
   |
81 |     license = licenses.gpl3;
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/clight/default.nix:56:5:
```
   |
56 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112254` at 6e1d9046 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/1c2b9014f15a8ff4bf99a4a5a3791fac">clight</a></li></li>
    <li><a href="https://gist.github.com/1283f64b4cce115025b23ae010949825">clightd</a></li></li>
  </ul>
</details>
<details>
  <summary>2 warnings produced:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/clight/clightd.nix:81:5:
```
   |
81 |     license = licenses.gpl3;
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/clight/default.nix:56:5:
```
   |
56 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112254` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>clight</li>
<li>clightd</li>
</ul>
</details>

**Review by lovesegfault — APPROVED** [review]

**Review by cole-h — APPROVED** [review]
Diff LGTM, thanks!

---

Result of `nixpkgs-review pr 112254` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>clight</li>
    <li>clightd</li>
  </ul>
</details>

**Pull request merged**